models/Report.ts

```diff
@@ -0,0 +1,7 @@
+export type CatSterilizationReport = {
+  content: string;
+};
+
+export const castToReport = (
+  backendReport: string
+): CatSterilizationReport => ({ content: backendReport });
```

Comment:
This is probably not necessary, but the whole app uses this template/pattern of `castToX` functions. Using `castToReport` should keep this "standard" consistent across the whole app. Also, we might process the `backendReport` parameter differently in the future. Instead of `{ content: backendReport }` we could return an object with many other keys containing more specific values that are typical for reports, such that they can be typed and therefore easily accessed e.g. date range, number of cats (really anything else related to the report).